docs/guide/rpc-api.md

```diff
@@ -1,5 +1,309 @@
-# The Ethereum RPC API
+# RPC API
 
-MetaMask uses the `ethereum.sendAsync()` (and soon, `ethereum.request()`) API to wrap an RPC API which is based on an interface exposed by all Ethereum clients, with some extra methods that are provided by MetaMask, as a key-holding signer. You can look up how to pass these methods to the `window.ethereum` object [here](./Ethereum-Provider).
+MetaMask uses the [`ethereum.request(args)` method](./ethereum-provider.html#ethereum-request-args) to wrap an RPC API.
 
-For the full API, please see [EIP 1474](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1474.md).
```

Comment:
Now that the title of this page has changed, it no longer seems appropriate to call this the "full" API. The Ethereum wiki link below documents a wider variety of RPC methods than this EIP does, and even that doesn't show them all (e.g. `eth_subscribe`)

Reply:
Maybe we can link nothing on this line, and link both below? 🤔 Not sure.